Makefile

```diff
@@ -2,12 +2,20 @@ GIT_BRANCH=`git rev-parse --abbrev-ref HEAD`
 USER=`whoami`
 STAGING_URL="https://docs-mongodborg-staging.corp.mongodb.com"
 PRODUCTION_URL="https://docs.mongodb.org/meta"
-STAGING_BUCKET=docs-mongodb-org-staging
+STAGING_BUCKET=docs-mongodb-org-prd-staging
 PRODUCTION_BUCKET=docs-mongodb-org-prod
```

Comment:
Shouldn't this be "docs-mongodb-org-prd"?

Reply:
updated